identity/manager.go

```diff
@@ -0,0 +1,70 @@
+// Maps Ethereum account to dto.Identity.
+// Currently creates a new eth account with PASSPHRASE password on CreateNewIdentity().
+
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/mysterium/node/service_discovery/dto"
+	"strings"
+)
+
+const PASSPHRASE = ""
+
+type identityManager struct {
+	keystoreManager keystoreInterface
+}
+
+func NewIdentityManager(keydir string) *identityManager {
+	return &identityManager{
+		keystoreManager: keystore.NewKeyStore(keydir, keystore.StandardScryptN, keystore.StandardScryptP),
+	}
+}
+
+func accountToIdentity(account accounts.Account) *dto.Identity {
+	identity := dto.Identity(account.Address.Hex())
+	return &identity
+}
+
+func identityToAccount(identityString string) accounts.Account {
+	return accounts.Account{
+		Address: common.HexToAddress(identityString),
+	}
+}
+
+func (idm *identityManager) CreateNewIdentity() (*dto.Identity, error) {
+	account, err := idm.keystoreManager.NewAccount(PASSPHRASE)
```

Comment:
PASSPHRASE -> should now be a parameter to `CreateNewIdentity()` or reuse `keydir` from `NewIdentityManager` factory